packages/backend/src/ee/services/AiAgentService.ts

```diff
@@ -565,6 +622,64 @@ export class AiAgentService {
         return { jobId, threadUuid };
     }
 
+    async generateAgentThreadResponseFromScheduler({
```

Comment:
I was a bit confused with naming before I figured out the difference.

How about:

`generateAgentThreadResponse` -> `scheduleGenerateAgentThreadResponse`

`generateAgentThreadResponseFromScheduler` -> `generateAgentThreadResponse`

Reply:
updated